src/util/ParseInputCoordinates.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function parseCornerCoordinateInputs(options,coord,callback) {
```

Comment:
This looks great. Now do you think we might make a unit test for this function, where we also describe what it does using a sentence? This could be quite simple, like this test:

https://github.com/publiclab/image-sequencer/blob/6028520197d8841e7b52c5b272ac187a4ea40ce6/test/modules/replace.js#L17-L25